src/OrleansRuntime/LogViews/GrainStateWithMetaData.cs

```diff
@@ -30,15 +36,27 @@ object IGrainState.State
             }
         }
 
+        /// <summary>
+        /// Constructor with TView
+        /// </summary>
+        /// <param name="initialview"></param>
         public GrainStateWithMetaDataAndETag(TView initialview)
         {
             StateAndMetaData = new GrainStateWithMetaData<TView>(initialview);
         }
+
+        /// <summary>
+        /// Default constructor
```

Comment:
Please use http://stylecop.soyuz5.com/SA1642.html. Also for short summaries, consider having them in a single line, as we mentioned in the past to avoid having more comments than code when it's not needed.